app/models/sandwich.rb

```diff
@@ -1,6 +1,7 @@
 class Sandwich < ActiveRecord::Base
+  BASE_PLAN = 'test'.freeze
```

Comment:
I think it's fine to use the plan name as the indicator for async / sync, but it might make sense for provisioning state to be kept on the sandwich instance as it feels like a more appropriate property for the add-on instance and matches our internal implementation.